00-HTML-CSS-basics/BasicsHTML/BasicsHTML4.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html> <!--Standard mode HTML5-->
+<html lang="en">
+<head>
+    <meta charset="UTF-8"> <!--Unicode character-->
+    <meta name="description" content="Bootcamp Web UI">
+    <meta name="keywords" content="HTML,CSS,JavaScript">
+    <meta name="author" content="Antonella Buono">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!--dimensions and scale for different devices-->
+    <title>Form</title>
+</head>
+<body>
+    <div class="container">
```

Comment:
Anto, consulta, capaz no me estoy dando cuenta, pero las classes donde se definieron? porque no veo un stylesheet vinculado ni tampoco classes declaradas dentro de un tag <styles>.

Reply:
Hola Esteban! este es el ejercicio The Basics of HTML el inciso VI, y no pedia incluir estilos, al menos yo entendi eso.